src/main/scala/au/com/agiledigital/toolform/util/ConfigHelpers.scala

```diff
@@ -0,0 +1,24 @@
+package au.com.agiledigital.toolform.util
+
+import pureconfig.error.{CannotConvert, ConfigValueLocation}
+
+import scala.reflect.ClassTag
+
+/**
+  * Extensions to PureConfig functionality.
+  */
+object ConfigHelpers {
+
+  /**
+    * Convert a `String => Either[String, T]` into a `String => Option[ConfigValueLocation] => Either` such that after application
+    * - `Right(t)` becomes `_ => Right(t)`
+    * - `Left(message)` becomes `location => Left(CannotConvert(value, type, message, location)`
+    */
+  def handleErrorWithEither[T](f: String => Either[String, T])(implicit ct: ClassTag[T]): String => Option[ConfigValueLocation] => Either[CannotConvert, T] =
```

Comment:
`CannotConvert` => `ConfigReaderFailure` ?

Reply:
Fixed in a77d0f4